jumpapp

```diff
@@ -373,7 +377,11 @@ get_active_workspace() {
 }
 
 activate_window() {
-    wmctrl -i -a "$1"
+    if [[ -n "$keep_workspace" ]]; then
```

Comment:
Let's keep option logic out of the functions below the `##### External Interfaces #####` line.

Instead, we can leave the `activate_window` logic as is. Maybe introduce a new `activate_and_move_window` function that calls `wmctrl -i -R "$1"`. Then we can either a) [update inline](https://github.com/mkropat/jumpapp/blob/d32191eb1907872c4c15a0f86c06d95190a9cfc1/jumpapp#L90-L91) the logic that calls `activate_window` to make it optionally call `activate_and_move_window` instead, or b) extract a new function `handle_activate_window` or some such that can make the calls. Feel free to alter any of the names (including the existing `activate_window`) to what you think makes sense, they're only suggestions.

Finally, once those changes are done, we only need to add a single [test](https://github.com/mkropat/jumpapp/blob/d32191eb1907872c4c15a0f86c06d95190a9cfc1/t/test_jumpapp). It would basically be the same as the [`it_calls_activate_window_when_window_found_by_pid`](https://github.com/mkropat/jumpapp/blob/d32191eb1907872c4c15a0f86c06d95190a9cfc1/t/test_jumpapp#L76-L85) test, but we could call it `it_calls_activate_and_move_window_when_-R_is_passed`, asserting that `activate_and_move_window` was called.